AwesomeProject/screens/Menu/Menu.screen.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {NavigationContainer} from '@react-navigation/native';
+import {Button, View, ScrollView} from 'react-native';
+import styles from '../Menu/Menu.styles';
+import AboutMe from '../AboutMe/AboutMe.screen';
+
+export default Screens = () => (
+  <NavigationContainer>
+    <Stack.Navigator initialRouteName="Screens">
+      <Stack.Screen name="Menu" component={MenuScreen} />
+      <Stack.Screen name="Alexey Kundro" component={DetailsScreen} />
+    </Stack.Navigator>
+  </NavigationContainer>
+);
+
+MenuScreen = ({navigation}) => (
+  <View style={styles.container}>
+    <ScrollView>
+      <View style={styles.main}>
+        <Button
```

Comment:
This is simple button that don't have any styles and it doesn't suit us. For adding button like we discussed find another components in react-native for this.